packages/components/psammead-consent-banner/src/__snapshots__/index.test.jsx.snap

```diff
@@ -6,19 +6,24 @@ exports[`ConsentBanner should correctly render for ltr service 1`] = `
   font-weight: 400;
   font-style: normal;
   background-color: #323232;
-  padding: 1rem 0.5rem;
+  border-top: solid 0.125rem transparent;
```

Comment:
@HarryVerhoef Does this equal 1 or 2px?

Reply:
This should be equal to 1px to be consistent with the rest of the website design...

Reply:
Thanks for the comments Catharine,

0.125rem is 2px, I will change this to 0.0625rem (1px), sorry I thought that the transparent borders were 2px. Just checking, the link hover/focus transparent border is 2px right?.

Reply:
Correct, thank you